app/helpers/application_helper.rb

```diff
@@ -25,7 +25,7 @@ def govuk_button_link_to(body, url, html_options = {})
 
   def form_with(*args, &block)
     options = args.extract_options!
-    defaults = { html: { novalidate: true } }
+    defaults = { html: { novalidate: true, autocomplete: :off } }
```

Comment:
Is this ` autocomplete: :off` left over from testing?

Reply:
No, the JS expects it. `form.getAttribute('autocomplete') === 'off'`

Reply:
Ah understood - it looks like we're expecting the autocomplete attribute to work on forms by default.